docs/providers.md

```diff
@@ -147,3 +147,37 @@ There are two required labels for OpenShift to run an Atomic App.
   * run the container to allow the image to determine what OpenShift objects are created.
 1. `io.openshift.generate.token.as=env:TOKEN_ENV_VAR`
   * run the container with user token so `oc` commands can be run on behalf of the user from within the container pod.
+
+
+
+### Marathon
+
+**Features**
+
+The Marathon provider will deploy an application into Mesos cluster using Marathon
+scheduler.
+
+This provider requires configuration (`providerapi`) to be able to connect to Marathon API. If no `providerapi` is specified it will use `http://localhost:8080` as
+Marathon API url.
```

Comment:
Need docs on this, how do we pass `providerapi` when it's in `constants.py`? We can provide this in `answers.conf` yes?